src/main/resources/configForReports.conf

```diff
@@ -1,5 +1,5 @@
-report_path=twitter-analytics/reporting/ActiveUsers
+activeUserReport_path=twitter-analytics/reporting/activeUsers
```

Comment:
> configForReports.conf

could you clarify, please: Is your idea to store define configs for different repots in the same file?

Reply:
Yes, this file is supposed to contain configuration for all different report scripts.

Reply:
ok, thanks for clarifying.